libnavigation-ui/src/main/java/com/mapbox/navigation/ui/ConnectivityStatusProvider.java

```diff
@@ -9,9 +9,6 @@
 
 import java.util.HashMap;
 
-/**
- * TODO should be removed, logic moved to HybridRouter
- */
```

Comment:
not completely. Like we are still using the `public boolean isConnectedFast() {` somewhere but `NetworkStatusService` doesn't have one. Also the `NetworkStatusService` is written in a completely Coroutine way. We need to call it from Java.
Since `ConnectivityStatusProvider` is working as expected now, I would suggest to keep it and migrate to something new in 2.0

Reply:
Should we leave a comment so we don't forget?